Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/tracing/common/BaseTracingState.kt

```diff
@@ -29,10 +29,8 @@ abstract class BaseTracingState {
         return if (tracingStatus != GeneralTracingStatus.Status.TRACING_INACTIVE) {
             when (riskLevelScore) {
                 RiskLevelConstants.INCREASED_RISK -> R.color.colorSemanticHighRisk
-                RiskLevelConstants.UNKNOWN_RISK_OUTDATED_RESULTS,
-                RiskLevelConstants.NO_CALCULATION_POSSIBLE_TRACING_OFF -> R.color.colorSemanticUnknownRisk
                 RiskLevelConstants.LOW_LEVEL_RISK -> R.color.colorSemanticLowRisk
-                else -> R.color.colorSemanticNeutralRisk
+                else -> R.color.colorSemanticUnknownRisk
```

Comment:
What do we need this for?

Reply:
Opposite to what the names suggest, the colorSemanticNeutralRisk is the grey color used for the grey card, and colorSemanticUnknownRisk is white. The fallback color was grey before and I've now changed it to white since there won't be a grey card anymore.
Maybe these color names should be interchanged but I didn't want to introduce more unnecessary modifications for now.